package.json

```diff
@@ -94,7 +96,12 @@
     "test:integration-headless": "MOZ_HEADLESS=1 wdio tests/integration/wdio.conf.js",
     "test:integration-headless-ci": "MOZ_HEADLESS=1 ERROR_SHOTS=1 wdio tests/integration/wdio.conf.js",
     "test:integration-docker": "MOZ_HEADLESS=1 wdio tests/integration/wdio.docker.js",
-    "test": "npm run test:db:migrate && npm run test:tests && npm run test:coveralls"
+    "test": "npm run test:db:migrate && npm run test:tests && npm run test:coveralls",
+    "allure:generate": "npx allure generate ./allure-results --clean",
+    "allure:open": "npm run allure:generate && npx allure open ./allure-report",
+    "test:playwright": "npx playwright test tests/e2e/specs/monitor-landing.spec.js",
```

Comment:
Also, I think this means only the _one_ file will be run (and not monitor-e2e.spec.js, monitor-home.spec.js, or monitor-preference.spec.js). 🤷 
Maybe not an issue, but maybe worth calling out.